api/logs/views.py

```diff
@@ -220,7 +221,7 @@ class NodeLogContributors(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin,
     required_read_scopes = [CoreScopes.USERS_READ]
     required_write_scopes = [CoreScopes.NULL]
 
-    serializer_class = UserSerializer
+    serializer_class = UnregisteredContributorSerializer
```

Comment:
The reason why UserSerializer was being used instead of the NodeContributor serializer in the first place was that we just wanted the name of the contributor.  It was the simplest serializer.

This is for a very specific endpoint - `logs/{log_id}/contributors`. It is fetching the contributors that were added as a part of this log action (contributors_added).  If this log has nothing to do with adding contributors, it will return an empty list.    The whole reason behind this endpoint even existing was that we needed the contributors' names that were added in the My Projects page to formulate the logs, without needing an additional request. 

In summary - to use Node Contributor serializer I need to return the rest of the info about the contributor in this view https://github.com/CenterForOpenScience/osf.io/blob/develop/api/nodes/views.py#L539-543.  That seems like a lot of information for just needing the user's name.

Alternatively, we're returning a bunch of things in log params now - maybe we could get the names of the users and return them in params and do away with this endpoint altogether.